group_vars/betabarrel_cluster/vars.yml

```diff
@@ -55,6 +56,28 @@ nameservers: [
   '8.8.4.4',  # Google DNS.
   '8.8.8.8',  # Google DNS.
 ]
+network_private_management_id: "{{ stack_prefix }}_internal_management"
```

Comment:
```suggestion
network_private_management_id: "vlan983"
```

Reply:
To answer all the ^ questions here:

when the BB variables were created, the `network_private_management_id` was not added. And playbook failed. So I added it and then it failed for missing IP's. So I changed `ip_addresses.yml` to to match. And therefore (the answer to your question) => the naming confusion.
But you are right and the naming should be `vlanXXX` and not `bb_internal_managment` ...